src/extended_json.ts

```diff
@@ -234,6 +234,13 @@ function serializeValue(value: any, options: EJSONSerializeOptions): any {
     return { $numberDouble: Object.is(value, -0) ? '-0.0' : value.toString() };
   }
 
+  if (typeof value === 'bigint') {
+    if (!options.relaxed) {
+      return { $numberLong: BigInt.asIntN(64, value).toString() };
+    }
+    return Number(BigInt.asIntN(64, value));
```

Comment:
Let's add examples of this to the design doc so we can be sure we're all aligned on the choice we made here.